syncserver/src/tokenserver/extractors.rs

```diff
@@ -748,9 +740,9 @@ mod tests {
         let req = TestRequest::default()
             .data(state)
             .data(Arc::clone(&SECRETS))
-            .header("authorization", "Bearer fake_token")
-            .header("accept", "application/json,text/plain:q=0.5")
-            .header("x-keyid", "0000000001234-qqo")
+            .append_header(("authorization", "Bearer fake_token"))
```

Comment:
I think we're ok with using `append_header` in this case, since there's probably no existing header to add the value to, but we might want to swap this with `insert_header` to override any existing ones.